lib/bloc/pos_catalog/repository.dart

```diff
@@ -13,6 +13,7 @@ abstract class Repository {
   Future<Item> fetchItemByID(int id);
   //This should be converted later to support pagination.
   Future<List<Item>> fetchAllItems();
+  Future<List<Item>> fetchItemsByFilter(String filter);
```

Comment:
Isn't it better to consolidate these two interface functions to one with optional filter?
`fetchItems({Sting filter}`

Reply:
[Done](https://github.com/breez/breezmobile/pull/298/commits/abecca857f6d8dc694c3b74dc617f434a1906501)
addresses renaming comments too.